Docs/Features/Rule-Matching.md

```diff
@@ -1,42 +1,48 @@
-## About
-### What are Rules ?
-They are text strings, which when configured can automatically open a browser for you, instead of prompting for selecting a browser.. Thus saving a extra step. Each time, you open a link in Hurl, It goes through all the configured rules to see if any matches with link and if it matches, It automatically opens the associated Browser.
+# About rules
 
-### What about Rulesets ?
-Each Ruleset is a collection of rules. You can have as many rules as you want inside a Ruleset. Each Ruleset is associated to a Browser and You can have multiple rulesets for same browser. Additionally, Rulesets prioritized as per the ordering. If same rule exists in two different rulesets, the ruleset that's higher in the list is triggered first.
+## What are rules?
+
+They are text strings, which when configured can automatically open a browser for you, instead of prompting for selecting a browser. Thus saving an extra step. Each time you open a link in Hurl, it goes through all the configured rules to see if any matches with the URL and if it matches, it automatically opens the associated browser.
```

Comment:
```suggestion
They are text strings, which when configured can automatically open a browser for you, instead of prompting to select a browser. Thus saving a step. Each time you open a link via Hurl, it searches through all the configured rules to see if any matches with the URL and if it matches, it automatically opens the associated browser.
```